_posts/2023-02-08-.md

```diff
@@ -0,0 +1,86 @@
+---
+title: M&S's Rollercoaster Component Library Journey
+published: true
+author: Dilesh Mistry, Emma Patricios
+date: 2023-02-08
+categories:
+  - ui library
+tags:
+  - ui library
+  - design
+  - javascript
+banner:
+  image: digital-design-language.png
+---
+
+# M&S's Rollercoaster Component Library Journey
+
+Software Engineering at M&S have been developing the UI since 2014 and we have tried a few times to react to customer needs quickly over the years. This article gives an overview of our journey and the lessons we’ve learnt – from custom-built vanilla JS to an industry-standard, reusable framework.
+
+## How it started
+
+We started our headless commerce journey with an Angular framework we called FEAR (Front-End Architecture, internally it is a running joke that M&S loves a good acronym) and as the industry moved towards NodeJS and Handlebars we followed suit. It drew inspiration from eBay’s front-end framework which prioritised SEO and page speed.
+
+We named this new product FESK (Front-End Starter Kit). It contained everything an engineer needed to develop and deploy a web application, leveraging a pattern library and common utilities to solve customer problems faster than FEAR, using Vanilla JS.
```

Comment:
Maybe even mention FESK  PDP uses Incremental DOM - an alternative to VirtualDOM (React).